x/faucet/client/cli/tx.go

```diff
@@ -0,0 +1,80 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/Sifchain/sifnode/x/faucet/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+	faucetTxCmd := &cobra.Command{
```

Comment:
I think this would be a good place for a chainID guard too. ie != `sifchain`

Reply:
Added